src/models/sensor-definitions.ts

```diff
@@ -10,6 +10,18 @@ export interface ISensorStrings {
   [key:string]: string|IStringMap;
 }
 
+export const SpecialMeasurementUnits: IStringMap = {
+  "Wind Speed": "m/s_WS",
+  "Wind Direction": "°_WD",
+  "Wind Chill": "°C_WC",
+  "Heat Index": "°C_HI",
+  "Dew Point": "°C_DP",
+  "Relative Humidity": "%_RH",
+  "Station Pressure": "mbar_SP",
+  "Barometric Pressure": "mbar_BP",
+  "Altitude": "m_AL"
+}
+
```

Comment:
I think this should go into sensor-gdx-manager since these names and units are specific to the gdx sensors.